src/core/lib/iomgr/error.c

```diff
@@ -191,15 +192,16 @@ void grpc_error_unref(grpc_error *err, const char *file, int line,
                       const char *func) {
   if (grpc_error_is_special(err)) return;
   gpr_log(GPR_DEBUG, "%p: %" PRIdPTR " -> %" PRIdPTR " [%s:%d %s]", err,
-          err->refs.count, err->refs.count - 1, file, line, func);
-  if (gpr_unref(&err->refs)) {
+          err->atomics.refs.count, err->atomics.refs.count - 1, file, line,
```

Comment:
Done. I called it on both parameters of the gpr_log call as I figure saving it as a temp variable defeats the purpose of the atomic load.